packages/amplify-codegen-e2e-tests/src/__tests__/build-app-swift.test.ts

```diff
@@ -42,6 +49,11 @@ describe('build app - Swift', () => {
       console.log(schemaText); // log so that ci does not timeout
       updateApiSchemaWithText(projectRoot, 'amplifyDatasource', schemaText);
       await generateModels(projectRoot, outputDir);
+      await generateStatementsAndTypes(projectRoot);
```

Comment:
The Swift compile assertion was removed when moving from CircleCI to codebuild. This test only asserts on the GraphQL statements now. 

https://github.com/aws-amplify/amplify-codegen/commit/b435d50818e9b995a770f109a7e79a1d1cd430ab

This was probably done because there is no MacOS instances on codebuild.